gui_qt/games_registry_panel.py

```
@@ -461,21 +509,95 @@ def __init__(
+        self._workspace_body.setVisible(has_workspace)
+        self._switch_btn.setEnabled(False)
+        if not has_workspace:
+            self._apply_interaction_gate()
+
+    def _choose_workspace(self) -> None:
+        if self._host_task_running or self._is_registry_task_running():
+            return
+        start_dir = str(self._workspace_root or Path.home())
+        directory = QFileDialog.getExistingDirectory(
+            self,
+            "选择工作区根目录（存放 Game_* 与 games_registry.json）",
```

Ruff (0.15.21)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)